jupyter_elfinder/elfinder.py

```diff
@@ -1113,10 +1103,10 @@ def __info(self, path: str) -> Info:
                         tmb_url = self.__path2url(tmb)
                         info["tmb"] = tmb_url
                     else:
-                        # FIXME: This is not part of api 2.1
-                        self._response[R_TMB] = True
                         if info["mime"].startswith("image/"):
                             info["tmb"] = 1
```

Comment:
In the docs it says that this should be a string `"1"`. Ie not an integer.

https://github.com/Studio-42/elFinder/wiki/Client-Server-API-2.1#information-about-filedirectory

Reply:
Yes, noticed that, but I checked the javascript code, it mentioned `info.tmb != 1`: https://github.com/imjoy-team/jupyter-elfinder/blob/bdc257450f3326a7fee2949868bf84b101478086/jupyter_elfinder/static/js/elfinder.full.js#L15501

However, in JS, `"1" == 1` is true, `"1" ===1` is false. it means both will work (`info["tmb"] = 1` and `info["tmb"] = "1"`).

Let's stick with the spec then, will change that.

Reply:
Yes, noticed that, but I checked the javascript code, it mentioned `info.tmb != 1`: https://github.com/imjoy-team/jupyter-elfinder/blob/bdc257450f3326a7fee2949868bf84b101478086/jupyter_elfinder/static/js/elfinder.full.js#L15501

However, in JS, `"1" == 1` is true, `"1" ===1` is false. it means both will work (`info["tmb"] = 1` and `info["tmb"] = "1"`).

Let's stick with the spec then, will change that.

Reply:
Yes, noticed that, but I checked the javascript code, it mentioned `info.tmb != 1`: https://github.com/imjoy-team/jupyter-elfinder/blob/bdc257450f3326a7fee2949868bf84b101478086/jupyter_elfinder/static/js/elfinder.full.js#L15501

However, in JS, `"1" == 1` is true, `"1" ===1` is false. it means both will work (`info["tmb"] = 1` and `info["tmb"] = "1"`).

Let's stick with the spec then, will change that.

Reply:
Yes, noticed that, but I checked the javascript code, it mentioned `info.tmb != 1`: https://github.com/imjoy-team/jupyter-elfinder/blob/bdc257450f3326a7fee2949868bf84b101478086/jupyter_elfinder/static/js/elfinder.full.js#L15501

However, in JS, `"1" == 1` is true, `"1" ===1` is false. it means both will work (`info["tmb"] = 1` and `info["tmb"] = "1"`).

Let's stick with the spec then, will change that.